test/test_datapipe.py

```diff
@@ -1813,6 +1830,16 @@ def test_shuffler_mapdatapipe(self):
         shuffler_dp = input_dp1.shuffle()
         self.assertEqual(10, len(shuffler_dp))
 
+        # Serialization Test
+        shuffler_dp = input_dp1.shuffle()
+        it = iter(shuffler_dp)
+        exp = []
+        for _ in range(2):
+            exp.append(next(it))
+        shuffler_dp_copy = pickle.loads(pickle.dumps(shuffler_dp))
+        exp.extend(list(it))
+        self.assertEqual(exp, list(shuffler_dp_copy))
```

Comment:
Similar to the last PR, this test needs an update as we expect fewer elements from the copy.